src/components/Table.js

```diff
@@ -81,12 +81,30 @@ const Table = ({ getSelectedPacket, packets, config }) => {
           setWindowStart(frameno);
           setWindowEnd(frameno + config.packetWindowSize);
         }
+        updateWindow(frameno);
```

Comment:
This call will happen on every packet.  This is not required. Take out the `frameno` out of the `for` loop initialize it to `-Infinity` so that it will never be greater than `windowEnd` if we don't enter the `if(packet)` condition in the for loop and then after the `for` loop is completed, call `maybeUpdateWindow` (or `updateWindow` for now.)